server/util.go

```diff
@@ -173,6 +173,9 @@ func CreateSDKLogger(ctx *Context, out io.Writer) (log.Logger, error) {
 	if ctx.Viper.GetString(flags.FlagLogFormat) == flags.OutputFormatJSON {
 		opts = append(opts, log.OutputJSONOption())
 	}
+	opts = append(opts,
+		log.ColorOption(ctx.Viper.GetBool(flags.FlagColorLogs)),
+		log.TraceOption(ctx.Viper.GetBool("trace"))) // cmtcli.TraceFlag
```

Comment:
Let's keep the comment as it wasn't super clear the flag was for stacktrace and not for the trace level